Source/Classes/Utils/ConstantsValues.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constants.swift
+//  MLCardDrawer
+//
+//  Created by Pedro Silva Dos Santos on 07/11/22.
+//
+
+import Foundation
+
+struct ConstantsValues {
+    static let WIDTH: CGFloat = 100
```

Comment:
Tem uma convenção sobre nome de constante que:
_Names of types and protocols are UpperCamelCase. Everything else is lowerCamelCase._
Ou seja, exceto tipo e protocolos, os demais devem seguir o padrão lowerCamelCase.
Aqui no MELI, adotamos esse padrão UPPER_CASE?

Reply:
feito